bin/cluster_dev.sh

```diff
@@ -23,15 +23,27 @@ fi
 
 INNER_SHELL=${SHELL:-"/bin/bash"}
 
+echo "----- Faasm development cluster -----"
+
 docker-compose -f docker-compose.yml \
     up \
     --no-recreate \
     -d \
-    faasm-cli
+
+if [[ "$1" == "faasm" ]]; then
+    CLI_CONTAINER="faasm-cli"
+elif [[ "$1" == "cpp" ]]; then
+    CLI_CONTAINER="cpp"
+else
+    echo "Not starting a CLI container"
+    exit 0
+fi
```

Comment:
Acts a bit like `cli.sh` for the development cluster. Although they're similar, it's good to keep the two separate as the dev cluster is much more like a "real" cluster while the development set-up is very inter-connected with a lot of mounts.